minitap/mobile_use/sdk/types/platform.py

```diff
@@ -0,0 +1,78 @@
+from typing import Any, Literal
+from pydantic import BaseModel, ConfigDict, Field
+from pydantic.v1.utils import to_lower_camel
+from datetime import datetime
+
+
+TaskRunStatus = Literal["pending", "running", "completed", "failed", "cancelled"]
```

Comment:
Would something like this work?
```suggestion
TaskRunStatus = Literal[tuple(s.value.lower() for s in TaskStatus)]
```

The idea is that we are not DRY there, if `TaskStatus` change, `TaskRunStatus` won't. Problem: they are highly correlated